apps/spruce/src/pages/waterfall/VersionLabel/index.tsx

```diff
@@ -89,22 +106,28 @@ export const VersionLabel: React.FC<VersionFields> = ({
   );
 };
 
-const VersionContainer = styled.div`
+const VersionContainer = styled.div<{ size?: "small" | "default" }>`
   ${columnBasis}
 
-  > * {
-    font-size: 12px;
-    line-height: 1.3;
-  }
+  ${(props) => {
+    if (props.size === "small") {
+      return `
+          > * {
+            font-size: 12px;
+            line-height: 1.3;
+          }
+        `;
+    }
+  }}
 
   p {
     ${wordBreakCss}
   }
 `;
 
-const CommitMessage = styled(Body)`
+const CommitMessage = styled(Body)<{ trimMessage: boolean }>`
   display: -webkit-box;
   -webkit-box-orient: vertical;
   -webkit-line-clamp: 3;
-  overflow: hidden;
+  ${(props) => props.trimMessage && "overflow: hidden;"}
```

Comment:
I think you can potentially make all of these properties conditional on `trimMessage`; right now the overflow is incorrect
<img width="1624" alt="image" src="https://github.com/user-attachments/assets/b1d9b35c-e8b1-4182-82e2-bb3780c4b1a5">